wazo_calld/plugins/calls/services.py

```diff
@@ -288,21 +288,26 @@ def make_call_from_channel(ari, channel):
         return call
 
     @staticmethod
-    def make_call_from_ami_event(event):
-        event_variables = event['ChanVariable']
-        call = Call(event['Uniqueid'])
-        call.conversation_id = event_variables.get('CHANNEL(linkedid)') or None
-        call.status = event['ChannelStateDesc']
-        call.caller_id_name = event['CallerIDName']
-        call.caller_id_number = event['CallerIDNum']
-        call.peer_caller_id_name = event['ConnectedLineName']
-        call.peer_caller_id_number = event['ConnectedLineNum']
-        call.user_uuid = event_variables.get('WAZO_DEREFERENCED_USERUUID') or event_variables.get('XIVO_USERUUID') or None
-        call.dialed_extension = event_variables.get('WAZO_ENTRY_EXTEN') or None
+    def make_call_from_stasis_event(event):
+        channel = event['channel']
+        channel_variables = event['channel']['channelvars']
+        conversation_id = channel_variables.get('CHANNEL(linkedid)')
+        connected = channel.get('connected')
+        caller = channel.get('caller')
+        call = Call(conversation_id)
```

Comment:
I don't think that `CHANNEL(linkedinded)` is the same thing as `event['Uniqueid']`. This may explain why the integration tests did not pass.